### PR TITLE
Disable eventpipe test on OSX/x64

### PR DESCRIPTION
**Pull request by BruceForstall**
Also, add sections for all-arch OSX and OSX/arm64 for possible future use.

### CODE REVIEW

**Comment by BruceForstall** [comment]
Tracking: https://github.com/dotnet/runtime/issues/66174

**Comment by BruceForstall** [comment]
@tommcdon @agocke @fanyang-mono @hoyosjs @trylek Here is a proposed disable of eventpipe test on osx/x64 to address #66174. I triggered outerloop, where it's failed before. But failure isn't 100%.

**Review by tommcdon — APPROVED** [review]
Thank you Bruce!

File: src/tests/issues.targets [hunk 1]
```diff
@@ -659,6 +659,21 @@
...
+    <!-- OSX x64 on CoreCLR Runtime -->
+    <ItemGroup Condition="'$(XunitTestBinBase)' != '' and '$(TargetsOSX)' == 'true' and '$(TargetArchitecture)' == 'x64' and '$(RuntimeFlavor)' == 'coreclr' ">
+        <ExcludeList Include="$(XunitTestBinBase)/profiler/eventpipe/eventpipe/*">
+            <Issue>https://github.com/dotnet/runtime/issues/66174</Issue>
+        </ExcludeList>
+    </ItemGroup>
+
+    <!-- OSX arm64 on CoreCLR Runtime -->
+    <ItemGroup Condition="'$(XunitTestBinBase)' != '' and '$(TargetsOSX)' == 'true' and '$(TargetArchitecture)' == 'arm64' and '$(RuntimeFlavor)' == 'coreclr' ">
+    </ItemGroup>
```
Comment (hoyosjs) [inline src/tests/issues.targets:675]:
```suggestion
    <ItemGroup Condition="'$(XunitTestBinBase)' != '' and '$(TargetsOSX)' == 'true' and '$(RuntimeFlavor)' == 'coreclr' ">
        <ExcludeList Include="$(XunitTestBinBase)/profiler/eventpipe/eventpipe/*">
            <Issue>https://github.com/dotnet/runtime/issues/66174</Issue>
        </ExcludeList>
    </ItemGroup>

    <!-- OSX x64 on CoreCLR Runtime -->
    <ItemGroup Condition="'$(XunitTestBinBase)' != '' and '$(TargetsOSX)' == 'true' and '$(TargetArchitecture)' == 'x64' and '$(RuntimeFlavor)' == 'coreclr' ">
    </ItemGroup>

    <!-- OSX arm64 on CoreCLR Runtime -->
    <ItemGroup Condition="'$(XunitTestBinBase)' != '' and '$(TargetsOSX)' == 'true' and '$(TargetArchitecture)' == 'arm64' and '$(RuntimeFlavor)' == 'coreclr' ">
    </ItemGroup>
```

**Review by trylek — APPROVED** [review]
LGTM, thanks Bruce!

**Review by hoyosjs — COMMENTED** [review]
I just looked at the data again - it happens on both architectures

**Comment by hoyosjs** [comment]
Sorry I missed the issue - I took off the label because I was under the impression that it had been disabled.

**Pull request merged**